java/1.8/Dockerfile

```diff
@@ -19,6 +19,15 @@ RUN cd /usr/lib && \
   sbt about && \
   rm -f /tmp/sbt.zip
 
+ENV MAVEN_VERSION 3.6.2
+ENV MAVEN_HOME /usr/lib/apache-maven-$MAVEN_VERSION
+ADD http://mirror.reverse.net/pub/apache/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.zip /tmp/maven.zip
```

Comment:
For string interpolation, do you need curly braces?
https://docs.docker.com/engine/reference/builder/#environment-replacement

Reply:
In bash/shell scripting it's more a kind of best-practice than a rule.